### ISSUE TITLE
Please make a GTK version of this theme! (Feature Request)

### ISSUE THREAD

**Issue by Prayag2** [Luwx/Lightly#51]
This theme is awesome! It makes KDE look very modern and polished..however, there's no GTK theme for this and apps like pamac, evince and celluloid are all GTK based. They look quite different from qt apps with this theme so it would be nice if a gtk theme was available for the same! Thanks!

**Comment by Taza53** [comment]
Love the idea but would it be even possible because as far as I know you can't change the application theme without changing the colour and other settings(I have only used plasma).

**Comment by DiegoMagdaleno** [comment]
@Taza53 what do you mean?

**Comment by ryu-ketsueki** [comment]
> Love the idea but would it be even possible because as far as I know you can't change the application theme without changing the colour and other settings(I have only used plasma).
> 
> Edit: can you change gtk application theme without changing the colour scheme.

Yes you can, Breeze does that quite nicely. Only need to use the default Breeze GTK theme and change color schemes to see the changes happen in GTK side.

**Comment by summetdev** [comment]
Any progress on this? Really wanted to see this as GTK theme.

**Comment by zxq5-dev** [comment]
this style for gtk/gnome apps on kde would be so good

**Comment by Vistaus** [comment]
> this style for gtk/gnome apps on kde would be so good

Absolutely!

**Comment by anonyco** [comment]
This is my first time getting into theming but I'm going to see if I can patch Lightly together with https://github.com/L4ki/Breeze-Gently I'll be using this on an opensuse tumbleweed build system that lives as a chroot of an alpine-based instant-on WayFire-based kiosk that will use busybox to bootstrap a full graphical boot interface faster than it takes to load grub (by skipping both the bootloader and initramfs entirely!) Wish me luck

One thing of curiousity to me is DeepIn's unified qt+gtk solution of https://github.com/linuxdeepin/qt5integration
